### DISCUSSION TITLE
Working with MQTT milliseconds and display interface

### DISCUSSION THREAD

**Discussion by Silonei-Duarte** [SCADA-LTS/Scada-LTS#3028] [Ideas]
Good afternoon, dear all, I'm new here, and I'm from Brazil.

I've been developing an interface to capture MQTT data in SCADA_LTS, which is updated in milliseconds, since they are critical sensors.

However, the interface, both in the configuration and in the Datapoints display, only displays seconds. I realized that it is possible to collect data in milliseconds by adding a 0 in front, 0500 seconds,
although the interface then changes to 500 seconds, the data is still collected with reference to 0500

However, the interface in general still only displays in seconds; I understand that it goes beyond a configuration file. Can you tell me where to start to implement this change or if there are any plans to add milliseconds to the code in future native versions.

**Comment by Limraj** [comment]
Hi @Silonei-Duarte,
Currently missing this feature, I encourage you to create an issue to add the ability to configure this Data Source in milliseconds. It shouldn't require much work, so if you create such an issue, it should be included in the next version.

edit:
Okay, I see that the issue has already been added, thank you for your contribution, we will try to implement this change in the next version in 2.8.0 or 2.9.0.

Regards,
Kamil Jarmusik
